samples/rag-cosmosdb/README.md

```diff
@@ -13,6 +13,24 @@ The sample is available in the following language stacks:
 
 Please refer to the [root README](../../README.md#requirements) for common prerequisites that apply to all samples.
 
+### Chat Storage Configuration
+
+1. **Managed Identity - Assign Permissions**:
+
+   - Please refer to this two documents to assign permissions for cosmos db account to the managed identity.
+     - [Control Plane operations](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/security/how-to-grant-control-plane-role-based-access?tabs=built-in-definition%2Ccsharp&pivots=azure-interface-cli)
+     - [Data Plane operations](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/security/how-to-grant-data-plane-role-based-access?tabs=built-in-definition%2Ccsharp&pivots=azure-interface-cli)
```

Comment:
Could you please suggest what are the minimal role assignment requirements for this sample?
It will be confusing for the users to figure this out from all the roles available.